test/plugins/individual-collateral/lido/constants.ts

```diff
@@ -0,0 +1,19 @@
+import { bn, fp } from '../../../../common/numbers'
+
+// Mainnet Addresses
+export const RSR = '0x320623b8e4ff03373931769a31fc52a4e78b5d70'
```

Comment:
In general I don't see a reason to redeclare all this data. A lot of it is in `common/configuration` and we could just import from there and access the `networkConfig`

Reply:
I agree. The question is do we move all to `common/configuration` or only the repeated ones for now or the token addresses only (which are required for deployment script as well)  and not the "auxiliary" ones?

Reply:
I would keep the auxiliary ones here.